subprojects/snapshots/src/test/groovy/org/gradle/internal/fingerprint/overlap/impl/DefaultOverlappingOutputDetectorTest.groovy

```diff
@@ -21,31 +21,35 @@ import org.gradle.internal.file.FileMetadata.AccessType
 import org.gradle.internal.file.impl.DefaultFileMetadata
 import org.gradle.internal.fingerprint.FileCollectionFingerprint
 import org.gradle.internal.fingerprint.impl.AbsolutePathFingerprintingStrategy
+import org.gradle.internal.fingerprint.impl.DefaultCurrentFileCollectionFingerprint
 import org.gradle.internal.hash.HashCode
+import org.gradle.internal.snapshot.CompleteDirectorySnapshot
 import org.gradle.internal.snapshot.FileSystemSnapshot
+import org.gradle.internal.snapshot.MissingFileSnapshot
 import org.gradle.internal.snapshot.RegularFileSnapshot
 import spock.lang.Specification
+import spock.lang.Unroll
 
 class DefaultOverlappingOutputDetectorTest extends Specification {
     def detector = new DefaultOverlappingOutputDetector()
 
     def "detects no overlap when there are none"() {
-        def outputFilesAfterPreviousExecution = ImmutableSortedMap.<String, FileCollectionFingerprint>of(
+        def outputFilesAfterPreviousExecution = ImmutableSortedMap.<String, FileCollectionFingerprint> of(
             "output", AbsolutePathFingerprintingStrategy.INCLUDE_MISSING.emptyFingerprint
         )
-        def outputFilesBeforeExecution = ImmutableSortedMap.<String, FileSystemSnapshot>of(
+        def outputFilesBeforeExecution = ImmutableSortedMap.<String, FileSystemSnapshot> of(
             "output", FileSystemSnapshot.EMPTY
         )
         expect:
         detector.detect(outputFilesAfterPreviousExecution, outputFilesBeforeExecution) == null
     }
 
-    def "detects overlap when there is one"() {
+    def "detects overlap when there is a stale root"() {
         def staleFileAddedBetweenExecutions = new RegularFileSnapshot("/absolute/path", "path", HashCode.fromInt(1234), DefaultFileMetadata.file(0, 0, AccessType.DIRECT))
-        def outputFilesAfterPreviousExecution = ImmutableSortedMap.<String, FileCollectionFingerprint>of(
+        def outputFilesAfterPreviousExecution = ImmutableSortedMap.<String, FileCollectionFingerprint> of(
```

Comment:
Dunno, but IntelliJ does this for some reason. I took a quick look but couldn't find out which setting controls this. I won't fix because the next time it will be broken anyway.

Reply:
Removed.